src/parser/shaman/enhancement/modules/checklist/Module.tsx

```diff
@@ -29,6 +31,7 @@ class Checklist extends BaseChecklist {
   protected flametongue!: Flametongue;
   protected flametongueRefresh!: FlametongueRefresh;
   protected hailstorm!: Hailstorm;
+  protected lightningShield!: LightningShield;
```

Comment:
Looks like you added this to Module, but never added it to the Checklist? So this doesnt actually show in the checklist.